### ISSUE TITLE
HEAD fails to compile due to new_post_eip150() parameter count change

### ISSUE THREAD

**Issue by lhl** [ethcore/parity#3469]
It looks like this commit https://github.com/ethcore/parity/commit/528bf17af593b89508a4c333115daf942a3a7942 changed Schedule::new_post_eip150() to take 4 instead of 3 parameters.

This is not updated in ethcore/src/engines/authority_round.rs https://github.com/ethcore/parity/blob/1daba384cc57676a620b6545b7707a3b38cb0f73/ethcore/src/engines/authority_round.rs#L193

and causes compilation to fail.

```
error[E0061]: this function takes 4 parameters but 3 parameters were supplied
   --> ethcore/src/engines/authority_round.rs:193:3
    |
193 | 		Schedule::new_post_eip150(true, true, true)
    | 		^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: the following parameter types were expected:
    = note: usize, bool, bool, bool

error: aborting due to previous error

error: Could not compile `ethcore`.
```

Looks like it shows up in CI as well, but this must have been failing for the past 5 days?
https://travis-ci.org/ethcore/parity/jobs/176263619#L2875

**Comment by arkpar** [comment]
Fixed in #3470

**Comment by lhl** [comment]
Thanks for the fix, builds now!
